go/config.pb.go

```diff
@@ -2,7 +2,6 @@
 //
 // SPDX-License-Identifier: Apache-2.0
 
-
```

Comment:
I think this needs to be reverted? The next line indicates that this file was generated by `protoc-gen-go`. So, I think we should need to skip the `*.pb.go` files

Reply:
The change is reverted and files are skipped from linting check.